### PR TITLE
boards/z1: make use of Makefile.include.serial

### PR DESCRIPTION
**Pull request by thomaseichinger**
Use `Makefile.include.serial` and define `PORT_LINUX` and `PORT_DARWIN`.

### CODE REVIEW

File: boards/z1/Makefile.include [hunk 1]
```diff
@@ -16,12 +16,13 @@ export OBJCOPY = $(PREFIX)objcopy
...
-ifeq ($(strip $(PORT)),)
-    export PORT = /dev/ttyUSB0
-endif
 export FFLAGS = --z1 -I -c $(PORT) -r -e -p $(HEXFILE)
 export OFLAGS = -O ihex
-export TERMFLAGS += -p "$(PORT)"
+
+export PORT_LINUX ?= /dev/ttyUSB0
+export PORT_DARWIN ?= /dev/tty.SLAB_USBtoUART
+export BAUD = 115200
```
Comment (haukepetersen) [inline boards/z1/Makefile.include]: no need, it's the default...

**Comment by haukepetersen** [comment]
give me the board and I will ack as soon as I tried this...

**Comment by thomaseichinger** [comment]
Comment addressed and squashed right away.

**Comment by haukepetersen** [comment]
Works as expected -> ACK and go (as Travis does not care about this change...)

**Pull request merged**